mrp_subcontracting_inhibit/README.rst

```diff
@@ -0,0 +1,95 @@
+==========================
+Mrp subcontracting inhibit
+==========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fmanufacture-lightgray.png?logo=github
+    :target: https://github.com/OCA/manufacture/tree/14.0/mrp_subcontracting_inhibit
+    :alt: OCA/manufacture
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_subcontracting_inhibit
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/129/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to inhibit subcontracting from purchase order according to a checbox defined at rule level that is passed on to the purchase order.
```

Comment:
```suggestion
If a product has a bill of materials (BoM) of subcontracting type, and the
vendor is included in the list of subcontractors of such BoM, when doing a
purchase order for such vendor and product, it will always generate a
subcontracting order.

This module allows to inhibit such behavior, acting as a regular purchase, as
this may not be convenient in some cases, as on manufacturing world, it's
frequent that your vendor sells you directly the same product without you
providing any raw material depending on lead times, material availability, etc.

A check is provided at purchase order line level for easily disable the
subcontracting part, as well as the same check at stock rule for propagating
the preference from replenishment option.
```